src/main/java/hudson/plugins/ec2/ssh/EC2UnixLauncher.java

```diff
@@ -278,6 +323,19 @@ public boolean verifyServerHostKey(String hostname, int port, String serverHostK
         }
     }
 
+    private String getEC2HostName(EC2Computer computer, Instance inst) {
```

Comment:
If ip expected as one of returned values, then better call method getEC2HostAddr()